doc/ledctl.pod

```diff
@@ -324,6 +324,42 @@ Displays version of ledctl and information about the license and exits.
 Prints information (system path and type) of all controllers detected by
 ledmon and exits.
 
+=item B<-P> or B<--list-slots> B<--controller>=I<controller>
+
+Prints all slots for the controller. Slot definition depends on the controller
+and is unique for the controller.
```

Comment:
> I would like to ask, what purpose do you have for this feature and when slot identifier is exactly needed for you? I'm thinking about adding function which will parse output from "--get-slot" command to return only one field (e.g. command: "ledctl --get-slot --controller npem --device /dev/nvme3n1 slot", output: "10000:02:01.0").
Please let me know if it will be applicable for your usages.

As I mentioned in https://github.com/intel/ledmon/issues/97#issuecomment-1121261784 

>One of the enhancement requests for libStorageMgmt is when a device stops responding. In this case the kernel may remove the device which causes the device node to get removed. When this occurs we don't have a good way of correlating where the device was to illuminate the associated LED. It's similar to this issue: https://github.com/intel/ledmon/issues/81

Some users are considering storing the slot information when all the devices are present, so when one of them disappears they can use the slot information to control the LEDs.  If the slot identifier was always consistent this would be possible.  I'm wondering if the slot identifier could be composed of multiple pieces of information, eg. `controller_sn-slot_id-device_WWN`?

Any other suggestions for how to handle when a device just disappears and the device node is removed too?  The other ideas I had suggested:

1. Light up all slots which don't have an associated device?
2. Illuminate all the devices which you can talk to, thus a slot with
a drive and no LEDs is the bad one(s)?

Reply:
Slot identifier (for both, VMD and NPEM) is not persistent. It can be changed with machine reboot, but not only, kernel modifications can also change it. We couldn't rely on it. When user will list slots, he can see which ones have drives assigned to them (and their device nodes), and the empty ones.

> Any other suggestions for how to handle when a device just disappears and the device node is removed too? The other ideas I had suggested:

> Light up all slots which don't have an associated device?
> Illuminate all the devices which you can talk to, thus a slot with
> a drive and no LEDs is the bad one(s)?

I think it will be possible to add when API will be available ([#97 ](https://github.com/intel/ledmon/issues/97)). Then LED functions to set slot can be used with many ways, related to needs.